modules/gdscript/gdscript_parser.cpp

```diff
@@ -8373,6 +8373,10 @@ void GDScriptParser::_add_warning(int p_code, int p_line, const String &p_symbol
 }
 
 void GDScriptParser::_add_warning(int p_code, int p_line, const Vector<String> &p_symbols) {
+	if(GLOBAL_GET("debug/gdscript/warnings/exclude_addons").booleanize() && base_path.begins_with("res://addons/"))
```

Comment:
There should be a space after the `if`, and the opening bracket below should be inline here.